pynecone/var.py

```diff
@@ -748,3 +748,54 @@ def type_(self):
         if "return" in self.fget.__annotations__:
             return self.fget.__annotations__["return"]
         return Any
+
+
+class PcList(list):
```

Comment:
`PcList` sounds more reasonable to me than just `List` cuz it's a list object specifically for pcnecone. IMO, `List` is appropriate when it is just a list but with more generalised capabilities.

Reply:
Yup sounds good!

Reply:
Small nit, but `PcList` doesn't follow PEP-8 naming convention, which we should defer to when possible. Prefer `PCList`:

> *Note: When using acronyms in CapWords, capitalize all the letters of the acronym. Thus HTTPServerError is better than HttpServerError.*

Source: [PEP-8](https://peps.python.org/pep-0008/)

Reply:
Thanks for catching that! [fd7c722](https://github.com/pynecone-io/pynecone/pull/339/commits/fd7c722759bc0e45631f10030e5446935b2f5df3)